### PR TITLE
feat: added list component

### PR DESCRIPTION
**Pull request by GuptaEesh**
Added lists. It contains:

>Unordered
Ordered
List without bullets, numbers etc.
Inline List
Cards in list

list
list1
list2

### CODE REVIEW

File: components/list.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
+
+.list-container-type{ 
+    border: 1px solid var(--primary-200);
+    display:flex;
```
Comment (AnuragSahu11) [inline components/list.css]: could use pretifier for consistent spacing between property and value, display: flex

**Review by AnuragSahu11 — APPROVED** [review]
Everything seems fine , you can merge , just added a small suggestion

File: components/list.css (hunk 1 above)
Comment (GuptaEesh) [reply components/list.css]: Done✅

**Review by saadahmadhere — APPROVED** [review]
Code looks awesome to me

**Comment by GuptaEesh** [comment]
Got 2 reviews. Merging the branch now.

**Pull request merged**